azure/packages/azure-local-service/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluidframework/azure-local-service",
-	"version": "2.0.0-rc.2.0.0",
+	"version": "2.0.0-rc.3.0.0",
```

Comment:
Do we want to drop the .0.0 ?

Reply:
No, because our tools do not understand arbitrary version schemes. We are using this form because the tools understand it, and it allows us to release patches without constantly bumping main to a new version.